template/palette_info.css

```diff
@@ -3,18 +3,22 @@
 }
 
 .color_palette_item {
-  height: 18px;
+  height: 22px;
   width: 22px;
   cursor: pointer;
   float: left;
+  -moz-border-radius: 13px;
+  -webkit-border-radius: 13px;
+  border-radius: 13px;
+  border: solid 2px black;
```

Comment:
Maybe it would be better to add some transparency to this border? How about `rgba(0, 0, 0, 0.7)`?